src/hooks/useCreateRoom.ts

```diff
@@ -13,40 +11,45 @@ export default function useCreateRoom(): () => Promise<void> {
 
     const sessionAccount = wallet?.address
 
-    const invite = useInviter()
+    const inviteSelf = useInviterSelf()
 
     const dispatch = useDispatch()
 
-    return useCallback(async () => {
-        if (!sessionAccount) {
-            throw new Error('Missing session account')
-        }
-
-        if (!account) {
-            throw new Error('Missing account')
-        }
-
-        if (!metamaskStreamrClient) {
-            throw new Error('Missing metamask streamr client')
-        }
-
-        const id = `${account}/streamr-chat/room/${uuidv4()}`.toLowerCase()
-        const roomName = getRoomNameFromRoomId(id)
-
-        const stream = await metamaskStreamrClient.createStream({
-            id,
-            partitions: 2,
-            description: roomName,
-        })
-
-        await invite({
-            invitees: [sessionAccount],
-            stream,
-        })
-
-        dispatch({
-            type: ActionType.AddRoomIds,
-            payload: [id],
-        })
-    }, [sessionAccount, metamaskStreamrClient, account, invite, dispatch])
+    return useCallback(
+        async (roomName: string) => {
+            if (!sessionAccount) {
+                throw new Error('Missing session account')
+            }
+
+            if (!account) {
+                throw new Error('Missing account')
+            }
+
+            if (!metamaskStreamrClient) {
+                throw new Error('Missing metamask streamr client')
+            }
+
+            const stream = await metamaskStreamrClient.createStream({
+                id: `/streamr-chat/room/${roomName}`.toLowerCase(),
```

Comment:
What if my room name has a white space in it? Gets url-encoded? Or what if the name I'm using already exists?

Reply:
The client rejects most special characters `!?,.+/&`, only allowing `-` and `_`. Whitespaces are also rejected.

Should we warn about this ("Please use alphanumeric characters, `-` and `_` only. The room id will get lowercased automatically") in the text below the romm name on the creation modal?